corehq/apps/registration/utils.py

```diff
@@ -51,15 +51,39 @@
 )
 
 
-def activate_new_user(form, created_by, created_via, is_domain_admin=True, domain=None, ip=None):
-    username = form.cleaned_data['email']
-    password = form.cleaned_data['password']
+def activate_new_user_via_reg_form(form, created_by, created_via, is_domain_admin=True, domain=None, ip=None):
     full_name = form.cleaned_data['full_name']
+    new_user = activate_new_user(
+        username=form.cleaned_data['email'],
+        password=form.cleaned_data['password'],
+        created_by=created_by,
+        created_via=created_via,
+        first_name=full_name[0],
+        last_name=full_name[1],
+        is_domain_admin=is_domain_admin,
+        domain=domain,
+        ip=ip,
+        atypical_user=form.cleaned_data.get('atypical_user', False),
+    )
+    return new_user
+
+
+def activate_new_user(
+    username, password, created_by, created_via, first_name=None, last_name=None,
+    is_domain_admin=True, domain=None, ip=None, atypical_user=False
```

Comment:
Will this ever be called without the arguments that have defaults? If not, it would make the code easier to read if the defaults were removed because you'd know that the value will always be passed in.